src/modalities/trainer.py

```diff
@@ -87,7 +94,11 @@ def train(
         model.train()
         cumulated_losses = self._reset_tracked_losses()
 
+        # throughput & MFU
         thoughput_aggregator = Aggregator[ThroughputAggregationKeys]()
+        theoretical_gpu_peak_performance = get_theoretical_gpu_peak_performance()
+        num_params = get_total_number_of_trainable_parameters(model)
```

Comment:
We need to fix before the computations of the number of parameters for hybrid sharding.

Reply:
done in #216